### PR TITLE
Add basic playground for all workflows examples

### PR DESCRIPTION
**Pull request by tico24**
Users who don't want to install k8s can now run all the examples in killercoda.

We pull them directly from git so no maintenance overhead on our part. We make the assumption that all the examples work (which they probably don't!)

You should be able to run this here: https://killercoda.com/tico24/course/argo-workflows/workflow-examples

### CODE REVIEW

File: argo-workflows/workflow-examples/examples.md
Comment (caelan-io) [inline argo-workflows/workflow-examples/examples.md]: typo: "kubecl" -> "kubectl" 

maybe add backticks to code format this as well

File: argo-workflows/workflow-examples/examples.md
Comment (caelan-io) [inline argo-workflows/workflow-examples/examples.md:8]: should we add a disclaimer here that some examples might not run properly? either a) because they are broken examples that need updating, or b) because of killercoda?

i'll defer to your judgment.

in either case, we can direct users to open an issue on our course repo i suppose.

File: argo-workflows/workflow-examples/intro.md
Comment (caelan-io) [inline argo-workflows/workflow-examples/intro.md:3]: should we say "(5 minutes max)" here?

**Review by caelan-io — APPROVED** [review]
Small comments only + 1 typo. Ready to merge once fixed!

File: argo-workflows/workflow-examples/intro.md
Comment (tico24) [reply argo-workflows/workflow-examples/intro.md:3]: The installer already echos ""It typically takes between 1m and 2m to get Argo Workflows ready." as soon as it starts.

**Pull request merged**
